fastn-core/src/auth/email_password.rs

```diff
@@ -551,13 +546,55 @@ async fn create_and_send_confirmation_email(
         .first(&mut conn)
         .await?;
 
+    // To use auth. The package has to have auto import with alias `auth` setup
+    let path = req_config
+        .config
+        .package
+        .eval_auto_import("auth")
+        .unwrap()
+        .to_owned();
+
+    let path = path
+        .strip_prefix(format!("{}/", req_config.config.package.name).as_str())
```

Comment:
Nopes. This must be `auth.<something>`. The content of email will be part of auth package, not current package.

Reply:
Create fastn-community.github.io/auth and switch to that to avoid confusion. We have the repo, just commit to it, why are you delaying it?

Reply:
End user will only add some auth package as a dependency in their FASTN.ftd file and will not add any other file in their package to get auth working end to end.